src/pages/ProfilePage.tsx

```diff
@@ -266,38 +247,31 @@ function ProfilePage(props) {
                         icon={Illustrations.ToddBehindCloud}
                         iconWidth={variables.modalTopIconWidth}
                         iconHeight={variables.modalTopIconHeight}
-                        title={props.translate('notFound.notHere')}
+                        title={translate('notFound.notHere')}
                         shouldShowLink
-                        link={props.translate('notFound.goBackHome')}
                     />
                 )}
             </View>
         </ScreenWrapper>
     );
 }
 
-ProfilePage.propTypes = propTypes;
-ProfilePage.defaultProps = defaultProps;
 ProfilePage.displayName = 'ProfilePage';
 
-export default compose(
-    withLocalize,
-    withOnyx({
-        personalDetails: {
-            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-        },
-        session: {
```

Comment:
Why have you removed `session` from `withOnyx` hook?

Reply:
Cause it was moved from the `ProfilePageOnyxProps`

Reply:
But where `session` param is now taken from?

Reply:
updated